Typography/src/helper-text/src/HelperText.styles.tsx

```diff
@@ -7,4 +7,12 @@ export const StyledHelperText = styled(BaseTypography, {
   fontStyle: 'italic',
   fontSize: '$2',
   lineHeight: '$4',
+
+  variants: {
+    variant: {
+      error: {
+        color: '$interactive-critical',
```

Comment:
Now that we expose a color prop, I'm not sure if a variant makes as much as a specialized component. Worth a discussion.